lms/djangoapps/api_manager/users/views.py

```diff
@@ -769,7 +770,15 @@ def post(self, request, user_id):
             default_cohort = get_cohort_by_name(course_key, CourseUserGroup.default_cohort_name)
         except CourseUserGroup.DoesNotExist:
             default_cohort = add_cohort(course_key, CourseUserGroup.default_cohort_name, CourseCohort.RANDOM)
-        add_user_to_cohort(default_cohort, user.username)
+        try:
+            add_user_to_cohort(default_cohort, user.username)
+        except AlreadyAddedToCohortException:
+            msg_tpl = _('Student {student} already added to cohort {cohort_name} for course {course}')
+            response_data = {
+                'message': msg_tpl.format(student=user.username3, cohort_name=default_cohort.name, course=course_key)
```

Comment:
@smarnach the issues with 500 you saw was caused by this.

@mtyaka if someone interested how did it get there - I was switching to console using Cmd+3 - probably I pressed `3` a bit before `Cmd` :)